README.md

```diff
@@ -1,6 +1,39 @@
-[![Build Status](https://travis-ci.org/BlueBrain/Livre.png?branch=master)](https://travis-ci.org/BlueBrain/Livre)
+# Livre
+![Livre](doc/images/livre_small.png)
```

Comment:
The image? It's already different from the one we had. Shall I use something else?